### PR TITLE
000 - Added build script to package.json

### PR DESCRIPTION
**Pull request by Adnanhoq**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (Leebee1392) [inline .vscode/launch.json:1]: Why do you need a launch.json?

**Review by sdagn1 — APPROVED** [review]

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
```
Comment (Leebee1392) [inline .vscode/launch.json:3]: If you need this file, please delete this

File: package.json [hunk 3]
```diff
@@ -1,19 +1,20 @@
 {
   "name": "nodejs-express-axios-starter",
   "version": "1.0.0",
-  "description": "",
+  "description": "## How to start the application 1. Run `npm install` to install application dependencies 2. Start the application with `npm start` or `npm run dev` to reload the server when changes are made 3. To check that your application is running enter url http://localhost:3000",
```
Comment (Leebee1392) [inline package.json:4]: This needs to go into a README

**Review by Leebee1392 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (Adnanhoq) [reply package.json:4]: Done

File: .vscode/launch.json (hunk 1 above)
Comment (Adnanhoq) [reply .vscode/launch.json:1]: Was for debugging, i'll delete for now

File: .vscode/launch.json (hunk 2 above)
Comment (Adnanhoq) [reply .vscode/launch.json:3]: Deleted

**Review by DylanS-03 — APPROVED** [review]

**Review by Leebee1392 — APPROVED** [review]

**Pull request merged**